lib/ansible/module_utils/redfish_utils.py

```diff
@@ -627,12 +641,15 @@ def get_boot_order(self):
         result["entries"] = boot_device_list
         return result
 
-    def set_bios_default_settings(self):
+    def get_multi_boot_order(self):
+        return self.aggregate(self.get_boot_order)
+
+    def set_bios_default_settings(self, systems_uri):
```

Comment:
This is a "set" method called from redfish_config (which doesn't pass in the systems_uri param). So need to remove that param here and just use self.system_uris[0] below.

Reply:
Done